frontend/src/scenes/urls.ts

```diff
@@ -173,6 +173,7 @@ export const urls = {
     dataWarehouse: (query?: string | Record<string, any>): string =>
         combineUrl(`/data-warehouse`, {}, query ? { q: typeof query === 'string' ? query : JSON.stringify(query) } : {})
             .url,
+    sqlEditor: (): string => `/sql`,
```

Comment:
Do we want this to be `/sql-editor`? Not that fussed either way, but it may be a bit clearer. Also, are we fine with using `SQL` and not `HogQL` everywhere? I think _yes_, but wanted to mention it

Reply:
temporary path just for testing purposes